lib/astarte_vmq_plugin/rpc/supervisor.ex

```diff
@@ -0,0 +1,40 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2025 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+defmodule Astarte.VMQ.Plugin.RPC.Supervisor do
+  @moduledoc """
+  Supervisor of the RPC server.
+  """
+
+  use Supervisor
```

Comment:
The default Supervisor from Elixir is used instead of Horde.Supervisor.
Just to confirm that this was the intended behavior.
So I reckon that each VerneMQ replica tries to start a `Astarte.VMQ.Plugin.RPC.Server` instance, but only one replica will be able to start it.
However, what if that replica dies, either because of a crash or because it is scaled down? Will another replica automatically start a new Server? I suspect not.